plans/tier1.fmf

```diff
@@ -240,6 +240,13 @@ adjust+:
             test+<:
                 - conversion-method/activation_key_conversion
 
+
+    /kernel-excluded-in-conf:
```

Comment:
Not an ask, just a rant.
We need to work on consistent naming, sometimes we use `-` dash sometimes `_` underscore as a separator :smiling_face_with_tear:

Reply:
Yes, I copied that from some name contained dashes. I'll change for all of them to have underscore, as it is more frequent